config/authorino/crd/authorino.3scale.net_authconfigs.yaml

```diff
@@ -0,0 +1,1040 @@
+
```

Comment:
I would automate the generation of this external resource. 

Check kuadrantctl how does this with makefile targets: https://github.com/Kuadrant/kuadrantctl/blob/ae6ee5f2c87bd10ec2053ad3b41826064d96e415/Makefile#L117
The makefile only needs one tag, and it will generate the resources in the repo.

Also a test to make sure nobody updates the auto-generated resources.